### PR TITLE
Added Parameter for choosing image and ssh-user

### PR DESCRIPTION
**Pull request by bokazio**
Added Parameter for choosing the OVH OS-Image and SSH-User from command line + Example for the Dokumentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,15 +60,26 @@ docker-machine create -d ovh node-1
 |``--ovh-consumer-key`` or ``$OVH_CONSUMER_KEY``            |Consumer Key      |none      |yes|
 |``--ovh-region``                                           |Cloud region      |GRA1      |no|
 |``--ovh-flavor``                                           |Cloud Machine type|vps-ssd-1 |no|
+|``--ovh-image``                                            |Cloud Machine image|Ubuntu 16.04 |no|
+|``--ovh-ssh-user``                                         |Cloud Machine SSH User|ubuntu |no|
```
Comment (yadutaf) [inline README.md:73]: I'd be tempted to mark this parameters as `(advanced) Cloud Machine ...` to avoid use by mistake.

File: README.md [hunk 2]
```diff
@@ -60,15 +60,26 @@ docker-machine create -d ovh node-1
...
 |``--ovh-project``                                          |Cloud Project name|single one|only if multiple projects|
 
-Note: OVH credentials may be supplied through arguments, environment or configuration file, by order
+Example with parameters:
```
Comment (yadutaf) [inline README.md]: Can you add a simple example first and then insert this new example as "advanced" ? Maybe be structure like

```
*Basic example*:

...

*Advanced example: Use CoreOS*

...

```

**Comment by yadutaf** [comment]
Neat ! I did not realize CoreOS was supported by Docker Machine, but apparently, it is (https://github.com/docker/machine/blob/master/libmachine/provision/coreos.go).

I did some suggestions to keep the README simple and guide as much as possible beginners toward the simple working use case.

**Comment by bokazio** [comment]
Yes, why not. I made some changes to the documentation to match your suggestions.

**Comment by yadutaf** [comment]
Awesome! Thanks.

**Pull request merged**